src/core_types.jl

```diff
@@ -1,14 +1,20 @@
 """
-    abstract Rotation{T} <: StaticMatrix{T}
+    abstract Rotation{N,T} <: StaticMatrix{T}
 
-An abstract type representing 3D rotations. More abstractly, they represent
-3×3 unitary (orthogonal) matrices.
+An abstract type representing `N`-dimensional rotations. More abstractly, they represent
+unitary (orthogonal) `N`×`N` matrices.
 """
-abstract Rotation{T} <: StaticMatrix{T}
+abstract Rotation{N,T} <: StaticMatrix{T}
 
-Base.size{R <: Rotation}(::Union{R,Type{R}}) = (3,3)
+typealias Rotation2{T} Rotation{2,T}
```

Comment:
Hmm. Funny I didn't think of that. I guess I'm used to the dimension parameter coming second, in which case only the typealias works. Will fix.